package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "restaurant_page",
+  "version": "1.0.0",
+  "description": "This is a restaurant landing page",
+  "main": "index.js",
+  "scripts": {
+    "test": "\"\""
+  },
```

Comment:
- [ ] __Necessary run common commands like `build`, `development` should be in `package.json` inside `scripts`. Here is a basic example__
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```